src/components/AgentDashboard(Rent)/ChangeStatus.js

```diff
@@ -394,7 +428,7 @@ function ChangeStatus() {
                 >
                   {/* for image */}
                   <div className="text-[1.2rem]">
-                    <b>Enter Details if Rented on B8R</b>
+                    <b>Enter Details if Rented of B8R</b>
```

Comment:
these should be "Rented on B8R" - @PrakharB8R, any reason you are replacing it? was this change approved?